pinax_theme_bootstrap/templates/account/base.html

```diff
@@ -16,5 +16,9 @@
     <a class="account-delete" href="{% url "account_delete" %}">
         {% trans "Delete account" %}
     </a>
+    {% comment "figure out how to detect other pinax apps and dynamically add here" %}{% endcomment %}
+    <a class="notice-settings" href="{% url "pinax_notifications:notice_settings" %}">
+        {% trans "Notices" %}
+    </a>
```

Comment:
@brosner There are a couple ways I can think of adding this capability.  One being a context processor that ships with the theme that does some `INSTALLED_APPS` inspection and provides a variable switch on.  But I wanted to check with you to see what you thought.